manifests/application.pvn.yaml

```
@@ -49,7 +57,7 @@ application:
       preconditions:
         - manualApproval: {}
         - releaseChannelStable:
-            releaseChannel: canary
+            releaseChannel: template_tester
```

this should be the name of the release channel

```suggestion
            releaseChannel: test_templates
```